docs/release/README.md

```diff
@@ -0,0 +1,134 @@
+# Release Guide
+The following guide was created to ease the release process and is
+aimed at being explanatory for both technical individuals as well as those with less development experience.
+
+# Table of Contents
+1) [Prerequsites](#prerequisites)
+    1) [Python](#python)
+    2) [Node](#node)
+    3) [Package Installers](#package-installers)
+2) [Target of Release](#target-of-release)
+3) [Release Candidate](#release-candidate)
+4) [Sprint Meeting Demonstration](#sprint-meeting-demonstration)
+    1) [Checking Out The Tag](#checking-out-the-tag)
+    2) [Ensuring Functional Software](#ensuring-functional-software)
+        1) [Back End](#back-end)
+        2) [Front End](#front-end)
+5) [Demonstrating](#demonstrating)
+6) [Releasing](#releasing)
+<a name="prerequisites"></a>
+## Prerequisites
+In order to make sure you can do everything necessary the following will have to be installed and available from the console:
+- `git`
+- `python`
```

Comment:
for `pysolr` we need `python < 3.8`

Reply:
@Dimfred I added a link to python3.7 in the python section